pkg/signature/kms/cliplugin/README.md

```diff
@@ -50,6 +50,23 @@ Exit status is ignored. Your program's stderr will be redirected to the main pro
 
 For authors using Go, we vend some helper functions to help you get started. See [handler](./handler/README.md)
 
+### Future Protocol Version
```

Comment:
I think this should be in an feature request rather than readme.

Reply:
It'll get more eyes as an issue, and we don't really have a roadmap for this library.